src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -66,6 +66,24 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 			}
 		}
 
+		[Test, Category ("UsesDevice")]
+		public void BuildBasicApplicationAndAotProfileIt ()
+		{
+			if (!HasDevices)
+				Assert.Ignore ("Skipping test. No devices available.");
+
+			var proj = new XamarinAndroidApplicationProject ();
+			var projDirectory = Path.Combine ("temp", TestName);
+			using (var b = CreateApkBuilder (projDirectory)) {
+				Assert.IsTrue (b.Build (proj), "Build should have succeeded.");
```

Comment:
Based on the documentation, `b.Build()` shouldn't be necessary, as `BuildAndStartAotProfiling` will implicitly perform a Build anyway.  Should this line be removed?